src/Endpoints/Delegates/HandlesSettings.php

```diff
@@ -421,4 +421,60 @@ public function resetEmbedders(): array
     {
         return $this->http->delete(self::PATH.'/'.$this->uid.'/settings/embedders');
     }
+
+    // Settings - Facet Search
+
+    /**
+     * @since Meilisearch v1.12.0
```

Comment:
do we really need this annotation (everywhere)?

Reply:
I wouldn't say we need it to the point where we should update all the code base. But if it shows up in the generated documentation, then it's useful information for the reader. What do you think?

Reply:
Ok